LanguageApp/app/screens/LearnerScenarioScreen.js

```diff
@@ -87,17 +97,14 @@ function LearnerScenarioScreen(translatorId) {
           <AppTitle style={styles.text}>{scenario.title}</AppTitle>
 
           <ScenarioImage uriLink={scenario.image} />
-          <Button
-            style={styles.button}
-            title={"Play prompt recording"}
-            onPress={playSound}
-          />
+
+          <AppButtonSecondary title={"Play prompt"} onPress={playSound} />
           <AppText style={styles.text}>{scenario.prompt}</AppText>
+
           <View style={styles.spacer}></View>
 
-          <Button
-            style={styles.button}
-            title={recording ? "Stop Recording" : "Start Recording Answer"}
+          <AppButtonSecondary
```

Comment:
Maybe we can avoid the duplicate code startRecording() and stopRecording() by making this a record button?